pkg/gce-pd-csi-driver/node_test.go

```diff
@@ -18,7 +18,6 @@ package gceGCEDriver
 import (
 	"context"
 	"fmt"
-	"io/ioutil"
```

Comment:
This is not the only place in the code base importing ioutil, should we change other places in the same PR?

Reply:
Others are under the vendor directory